src/Api/index.js

```diff
@@ -0,0 +1,63 @@
+import axios from 'axios';
+
+
+const instance = axios.create({
+    baseURL: 'http://localhost:3000'
+})
+
+
+instance.interceptors.request.use(function (config) {
+    console.log('Request sent successfully');
+    return config;
+}, function (err) {
+    console.log(err.status)
+    return Promise.reject(err);
+})
+
+instance.interceptors.response.use(function (response) {
+    console.log(response.status);
+    return response;
+}, function (err) {
+    console.log(err.message);
+    return Promise.reject(err);
+})
+
+export const callUserLoginApi = (enteredUsername, enteredPassword) => {
+    return instance.post('/login', {
+        username: enteredUsername,
+        password: enteredPassword,
+    });
+
+}
+
+export const callUserSignupApi = (enteredUsername, enteredPassword, enteredEmail) => {
+    return instance.post('/signup', {
+        username: enteredUsername,
+        password: enteredPassword,
+        email: enteredEmail
+    });
+}
+
+export const callCreateNoteApi = (id, title, description, date,) => {
+    return instance.post('/article', {
+        id: id,
+        title: title,
+        description: description,
+        date: date
+    });
+}
```

Comment:
Refactor advice. Request bodies are subject to change and hence it's a good idea to pass them as objects. Also in many cases the request bodies have >5 fields. It wouldn't be a good idea to pass each property as an argument.

Just do `callCreateNoteApi = (noteBody) => ....`